solidity/contracts/BondedECDSAKeepFactory.sol

```diff
@@ -29,7 +29,8 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 contract BondedECDSAKeepFactory is
     IBondedECDSAKeepFactory,
     CloneFactory,
-    AuthorityDelegator
+    AuthorityDelegator,
+    IRandomBeaconConsumer
```

Comment:
Aren't we missing import of `IRandomBeaconConsumer`?

Reply:
Oh I see, it gets imported in `import "@keep-network/keep-core/contracts/IRandomBeacon.sol";`.
Can we be explicit what we're importing? E.g.:
```solidity
import { 
    IRandomBeacon, 
    IRandomBeaconConsumer 
} "@keep-network/keep-core/contracts/IRandomBeacon.sol";.
```

Reply:
I'd love to but I am getting some truly bizarre output from truffle then. 
```
import {
    IRandomBeacon,
    IRandomBeaconConsumer
} from "@keep-network/keep-core/contracts/IRandomBeacon.sol";
``` 
```
/Users/piotr/go/src/github.com/keep-network/keep-ecdsa/solidity/contracts/BondedECDSAKeepFactory.sol:17:1: ParserError: Source "@keep-network/keep-core/contracts/IRandomBeacon.sol" not found: File import callback not supported
import {
^ (Relevant source part starts here and spans across multiple lines).

Compilation failed. See above.
Truffle v5.1.18 (core: 5.1.18)
Node v11.15.0
```

Do you know what could be happening here? I am out of ideas.

Reply:
I'm not sure why it doesn't work. Let's leave it as-is 🤷

Reply:
Yeah, I tried everything. I thought it's something about borked `build` artifacts or something not updating in `node_modules`. Tried all the combinations, and have the same error all the time.